contracts/HydraStaking/modules/VestedStaking/IVestedStaking.sol

```diff
@@ -1,13 +1,15 @@
 // SPDX-License-Identifier: MIT
 pragma solidity 0.8.17;
 
+import {IVesting} from "../../../common/Vesting/IVesting.sol";
+
 struct StakingRewardsHistory {
     uint256 totalReward;
     uint256 epoch;
     uint256 timestamp;
 }
 
-interface IVestedStaking {
+interface IVestedStaking is IVesting {
```

Comment:
why this is not IStaking as well?

Reply:
Well since Vesting is used only in VestedStaking/Delegation and before it was part of this contract, I thought it would be better to be in this interface (IVestedStaking), which is also included in the IStaking, so we should have all the functions.